### PR TITLE
AF-518: Docks locators in Library asset view

### PR DESCRIPTION
**Pull request by ederign**
@tomasdavidorg 

class for docks bars: qe-docks-bar-W (W is the side of the bar)

class for docks-items: qe-docks-item-W-org.kie.guvnor.explorer (W side, org.k Screen)

class for ListBar Header: qe-list-bar-header-CustomerSurvey-taskform.form---Others - format: qe-list-bar-header-file-type (I've replaced all " " to "-")

class for the content of the header:  qe-list-bar-content-CustomerSurvey-taskform.form---Others 

When there is no listbar, the main class is qe-static-workbench-panel-view

### CODE REVIEW

File: uberfire-workbench/uberfire-workbench-client-views-patternfly/src/main/java/org/uberfire/client/views/pfly/listbar/ListBarWidgetImpl.java [hunk 1]
```diff
@@ -219,6 +220,8 @@ public void addPart(final WorkbenchPartPresenter.View view) {
         parts.add(partDefinition);
 
         final FlowPanel panel = new FlowPanel();
+        setupCSSLocators(view,
```
Comment (paulovmr) [inline uberfire-workbench/uberfire-workbench-client-views-patternfly/src/main/java/org/uberfire/client/views/pfly/listbar/ListBarWidgetImpl.java]: @ederign I know that is a view, but would be nice to have a `verify` for this method's call on the `ListBarWidgetImplTest.java` methods that call `addPart`.

Comment (ederign) [inline uberfire-workbench/uberfire-workbench-client-views-patternfly/src/main/java/org/uberfire/client/views/pfly/listbar/ListBarWidgetImpl.java]: fixed.

**Comment by ederign** [comment]
E-mail to Run owners	E-mail to Service owners
	Test Result (1 failure / +1)
org.uberfire.java.nio.fs.file.SimpleUnixFileStoreTest.simpleTests

Not related.

**Comment by tomasdavidorg** [comment]
@ederign Thank you very much. This will really helps us to make tests more stable in the future.

**Pull request merged**